pytest.ini

```diff
@@ -0,0 +1,2 @@
+[pytest]
```

Comment:
add this line
DJANGO_SETTINGS_MODULE = Pro4U.settings

Reply:
@guybeckenstein done